### PR TITLE
Fix typo in "caractères"

### PR DESCRIPTION
**Pull request by noirbizarre**
Fixed translations and updated all translations (as some were missing).
(nb: as english is the fallback language, empty translations are filled with english on transifex pull)

### CODE REVIEW

File: udata/translations/fr/LC_MESSAGES/udata.po [hunk 1]
```diff
@@ -1330,27 +1274,17 @@ msgstr "Pagination"
...
-"previous and/or next page."
-msgstr ""
-"Vous n'avez pas à calculer vous-même les pages précédentes et suivantes\n"
-"puisque les URL sont disponible dans la réponse\n"
-"dans les attributs <code>previous_page</code> et <code>next_page</code>.\n"
-"Ils seront définis à <code>null</code> si il n'y a pas de page précédente"
-" et/ou suivante."
+"                <code>previous_page</code> and <code>next_page</code> attributes.\n"
+"                They will be set to <code>null</code> if there is no previous and/or next page."
+msgstr "Vous n'avez pas à calculer vous-même les pages précédentes et suivantes\npuisque les URL sont disponible dans la réponse\ndans les attributs <code>previous_page</code> et <code>next_page</code>.\nIls seront définis à <code>null</code> si il n'y a pas de page précédente et/ou suivante."
```
Comment (vinyll) [inline udata/translations/fr/LC_MESSAGES/udata.po]: _sont disponible**s**_
_via les attributs_ / _par les attributs_ (et _attributs_ vs _paramètres_)

File: udata/translations/fr/LC_MESSAGES/udata.po [hunk 2]
```diff
@@ -1362,29 +1296,19 @@ msgstr "Exemples"
...
 
 #: udata/templates/apidoc.html:97
 msgid "Verifying that httpie is working"
 msgstr "Vérifier que httpie fonctionne"
 
 #: udata/templates/apidoc.html:99
 msgid ""
-"Once httpie is installed, you can verify that it works as expected by "
-"typing that command in your shell:\n"
+"Once httpie is installed, you can verify that it works as expected by typing that command in your shell:\n"
```
Comment (vinyll) [inline udata/translations/fr/LC_MESSAGES/udata.po]: `<em>httpie</em>`

File: udata/translations/fr/LC_MESSAGES/udata.po [hunk 3]
```diff
@@ -1362,29 +1296,19 @@ msgstr "Exemples"
...
 "                "
-msgstr ""
-"Une fois httpie installé, vous pouvez vérifier qu'il fonctionne comme "
-"convenu en tapant cette commande dans votre terminal :"
+msgstr "Une fois httpie installé, vous pouvez vérifier qu'il fonctionne comme convenu en tapant cette commande dans votre terminal :"
```
Comment (vinyll) [inline udata/translations/fr/LC_MESSAGES/udata.po]: `<em>httpie</em>`

File: udata/translations/fr/LC_MESSAGES/udata.po [hunk 4]
```diff
@@ -1394,97 +1318,65 @@ msgstr "Cela doit retourner une réponse de ce style :"
 
 #: udata/templates/apidoc.html:133
 msgid ""
-"This is very verbose and we don't need all that information yet. That's "
-"why we use jq.\n"
+"This is very verbose and we don't need all that information yet. That's why we use jq.\n"
```
Comment (vinyll) [inline udata/translations/fr/LC_MESSAGES/udata.po]: `<em>jq</em>`

File: udata/translations/fr/LC_MESSAGES/udata.po [hunk 5]
```diff
@@ -1394,97 +1318,65 @@ msgstr "Cela doit retourner une réponse de ce style :"
...
 "                "
-msgstr ""
-"C'est très verbeux et nous n'avons pas besoin de toute cette information "
-"pour l'instant. C'est la raison pour laquelle nous utilisons jq."
+msgstr "C'est très verbeux et nous n'avons pas besoin de toute cette information pour l'instant. C'est la raison pour laquelle nous utilisons jq."
```
Comment (vinyll) [inline udata/translations/fr/LC_MESSAGES/udata.po]: `<em>jq</em>`

File: udata/translations/fr/LC_MESSAGES/udata.po [hunk 6]
```diff
@@ -1394,97 +1318,65 @@ msgstr "Cela doit retourner une réponse de ce style :"
...
 
 #: udata/templates/apidoc.html:136
 msgid "Verifying that jq is working"
 msgstr "Vérifier que jq fonctionne"
 
 #: udata/templates/apidoc.html:138
 msgid ""
-"Once jq is installed, you can verify that it works as expected by typing "
-"that command in your shell:\n"
+"Once jq is installed, you can verify that it works as expected by typing that command in your shell:\n"
```
Comment (vinyll) [inline udata/translations/fr/LC_MESSAGES/udata.po]: `<em>jq</em>`

File: udata/translations/fr/LC_MESSAGES/udata.po [hunk 7]
```diff
@@ -1394,97 +1318,65 @@ msgstr "Cela doit retourner une réponse de ce style :"
...
 "                "
-msgstr ""
-"Une fois jq installé, vous pouvez vérifier qu'il fonctionne en tapant "
-"cette commande dans votre terminal :"
+msgstr "Une fois jq installé, vous pouvez vérifier qu'il fonctionne en tapant cette commande dans votre terminal :"
```
Comment (vinyll) [inline udata/translations/fr/LC_MESSAGES/udata.po]: `<em>jq</em>`

Comment (noirbizarre) [inline udata/translations/fr/LC_MESSAGES/udata.po]: No, it's only translations, not html. If we want to had some markup, it should be in the template.
This page is particular. I had some work miving it to markdown allowing to have one file by language.

**Comment by noirbizarre** [comment]
OK, I just fixed the translation Transifex and reimported it.
Translations fixes should be in Transifex only, not as comment here (because they will be lost).
This PR only try to fix a visible mistakebefore going to production, to to improve wording style.
But, if you have time, don't hesitate to submit improvements on Transifex and submit another PR with thoses new translations 👍 

For the `<em>` markups, they shouldn't be in the translations (even for those already present). To me, the main problem is that a full page text is not translatable this way. I had started some work to move in in markdown files allowing to have one file by language.

**Comment by noirbizarre** [comment]
(or amend this PR with those new translations ;) )

**Comment by noirbizarre** [comment]
Maybe I need to close this one and wait for another PR when all strings have been reviewed and fixed or improvement in Transifex.

**Comment by davidbgk** [comment]
+1 to close it and amend directly on Transifex @vinyll

**Comment by noirbizarre** [comment]
Added to etalab/udata-gouvfr#141 to remember

**Pull request closed without merging**